src/test/java/edu/mit/dos/object/ObjectServiceIT.java

```diff
@@ -40,7 +51,7 @@ public void testPost() {
         map.add("title", "Tomb of Sitta Zubayda");
         HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<>(map, new HttpHeaders());
         String body = this.restTemplate.postForObject("/object",  request,String.class);
-        assertThat(body).isEqualTo("ok");
+        assertThat(body).isNotNull();
```

Comment:
In addition to testing that the response is not null, another line should test that the body contains the expected response.